rolling/distribution.yaml

```diff
@@ -2597,6 +2597,24 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: ros2-devel
     status: developed
+  master_board_sdk:
```

Comment:
This name is definitely too generic. If you read the name it would be expected that it would be a board for all use cases? I'd recommend a prefix `odri_` potentially or something related to the Solo Quadruped. 

Just reading the README opening "Hardware and Firmware of the Solo Quadruped Master Board."  It's optimal that the name capture that aspect. Maybe `solo_quad_master_board` or `odri_master_board` if it's intended to be more generic than the one robot.